pyvista/core/dataset.py

```diff
@@ -169,32 +257,104 @@ def active_tensors_name(self) -> str:
+        """Return the name of the active vectors array.
+
+        Examples
+        --------
+        Create a mesh, compute the normals, set them as active, and
+        return the name of the active vectors.
+
+        >>> import pyvista
+        >>> mesh = pyvista.Sphere()
+        >>> mesh_w_normals = mesh.compute_normals()
+        >>> mesh_w_normals.active_vectors_name = 'Normals'
+        >>> mesh_w_normals.active_vectors_name
+        'Normals'
+
+        """
         return self.active_vectors_info.name
 
     @active_vectors_name.setter
     def active_vectors_name(self, name: str):
-        """Set the name of the active vector."""
         self.set_active_vectors(name)
 
-    @property
-    def active_scalars_name(self) -> str:
-        """Return the active scalar's name."""
+    @property  # type: ignore
+    def active_scalars_name(self) -> str:  # type: ignore
+        """Return the name of the active scalars.
+
+        Examples
+        --------
+        Create a mesh, add scalars to the mesh, and return the name of
+        the active scalars.
+
+        >>> import pyvista
+        >>> mesh = pyvista.Sphere()
+        >>> mesh['Z Height'] = mesh.points[:, 2]
+        >>> mesh.active_scalars_name
+        'Z Height'
+
+        """
         return self.active_scalars_info.name
 
     @active_scalars_name.setter
     def active_scalars_name(self, name: str):
-        """Set the name of the active scalar."""
         self.set_active_scalars(name)
 
     @property
     def points(self) -> pyvista_ndarray:
-        """Return a pointer to the points as a numpy object."""
+        """Return a pointer to the points as a numpy object.
+
+        Examples
+        --------
+        Create a mesh and return the points of the mesh as a numpy
+        array.
+
+        >>> import pyvista
+        >>> cube = pyvista.Cube().clean()
+        >>> points = cube.points
+        >>> points
+        pyvista_ndarray([[-0.5, -0.5, -0.5],
+                         [-0.5, -0.5,  0.5],
+                         [-0.5,  0.5,  0.5],
+                         [-0.5,  0.5, -0.5],
+                         [ 0.5, -0.5, -0.5],
+                         [ 0.5,  0.5, -0.5],
+                         [ 0.5,  0.5,  0.5],
+                         [ 0.5, -0.5,  0.5]], dtype=float32)
+
+        Shift these points in the z direction and show that their
+        position is reflected in the mesh points.
+
+        >>> points[:, 2] += 1
+        >>> cube.points
+        pyvista_ndarray([[-0.5, -0.5,  0.5],
+                         [-0.5, -0.5,  1.5],
+                         [-0.5,  0.5,  1.5],
+                         [-0.5,  0.5,  0.5],
+                         [ 0.5, -0.5,  0.5],
+                         [ 0.5,  0.5,  0.5],
+                         [ 0.5,  0.5,  1.5],
+                         [ 0.5, -0.5,  1.5]], dtype=float32)
+
+        You can also update the points in-place.  For example, you can
+        scale the mesh with:
```

Comment:
Didactically this case seems the same as the previous one. Perhaps a case with `points[...] = new_points` would be better here?

Reply:
Agreed.